nautobot_ssot/integrations/aci/diffsync/adapters/aci.py

```diff
@@ -350,8 +351,21 @@ def load_devices(self):
                     )
                 )
                 model = device_specs["model"]
-            else:
+
+            if not model:
+                try:
+                    self.get("device_type", {"model": value["model"], "part_nbr": ""})
+                except ObjectNotFound:
+                    _devicetype = self.device_type(
+                        model=value["model"],
+                        manufacturer="Cisco",
+                        part_nbr="",
+                        u_height=1,
+                        comments="",
+                    )
+                    self.add(_devicetype)
```

Comment:
https://diffsync.readthedocs.io/en/latest/api/diffsync.html#diffsync.DiffSync.get_or_instantiate

```suggestion
                self.get_or_instantiate("device_type", ids={"model": value["model"], "part_nbr": ""}, attrs={"manufacturer": "Cisco", "u_height": 1, "comments": ""})
```

This should probably also work and be more concise

Reply:
you are the expert ;)